### ISSUE TITLE
Setting seeds for reproducibility

### ISSUE THREAD

**Issue by taknotts** [luyiyun/NormAE#8]
I have installed and am able to run NormAE on several metabolomics datasets.  One big problem is that the data is not reproducible- varies each time I run it, giving quite different results- RSDs range from 6 to 30 depending on run for the exact same data.  It is unfortunate since it does seem to normalize and reduce variance (most of the time).  I have tried setting every seed in python and relevant package- general random seed, numpy, PyTorch, and even limit threads- but have been unable to get reproducible results run to run.  Do you have any suggestions on how to resolve this issue?  Thanks!

**Comment by luyiyun** [comment]
Could you show your code about setting seeds?

**Comment by taknotts** [comment]
I am accessing python through R using the package reticulate.  So commands have the syntax to communicate to python through that package.  

`py_run_string("import random; random.seed(111)")`

`py_run_string("import numpy as np; np.random.seed(111)")`

`py_run_string("import torch; torch.manual_seed(111)")`
